.github/workflows/test.yml

```diff
@@ -22,4 +22,16 @@ jobs:
           key: v1/${{ runner.os }}/node-${{ matrix.node }}/${{ hashFiles('**/yarn.lock') }}
           restore-keys: v1/${{ runner.os }}/node-${{ matrix.node }}/
       - run: yarn
+      - name: Set up @percy/cli master
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          cd /tmp
+          git clone https://github.com/percy/cli
+          cd cli
+          yarn
+          yarn build
+          yarn global:link
+          cd ${{ github.workspace }} 
+          yarn remove @percy/cli && yarn link @percy/cli
```

Comment:
* just to confirm just yarn link `@percy/cli` links all the mono repo packages?
* update the PR description for why we're doing this, what it solves?

Reply:
Linking all packages now